### ISSUE TITLE
Default view when complex is turned on should just show the interactions in the near vicinity of the ligand, not the whole protein

### ISSUE THREAD

**Issue by fragalysis-reporter** [m2ms/fragalysis-frontend#213]
- Version: 0.3.1
- Name: Rachael
- Description: This is implemented by Matteo Ferla (matteo@well.ox.ac.uk) on his website michelNGLO:

code: https://github.com/matteoferla/MichelaNGLo-app

You can email him directly for help

![screenshot](https://github.com/m2ms/fragalysis-assets/blob/master/screenshot-1584923101694187199.png?raw=true)

**Comment by matteoferla** [comment]
In mine case I have the following code:

    protein.removeAllRepresentations();
    protein.addRepresentation("cartoon", {color: scheme, sele: '*'});
    protein.addRepresentation("hyperball", {color: scheme, sele: selection});
    const expanded = NGL.specialOps.expandSelection(protein, selection, radius);
    protein.addRepresentation("licorice", {sele: expanded});
    protein.addRepresentation("contact", { 
           masterModelIndex: 0,
            weakHydrogenBond: true,
            maxHbondDonPlaneAngle: 35,
            sele: expanded
        });

where the function that is a badly done monkey patch that is not yet changed to a prototype:

    NGL.specialOps.expandSelection = (protein, selection, radius) => {
            let selector = new NGL.Selection(selection.toString());
            const atomSet = protein.structure.getAtomSetWithinSelection(selector, parseFloat(radius));
            // expand selection to complete groups
            const atomSet2 = protein.structure.getAtomSetWithinGroup(atomSet);
            return `${atomSet2.toSeleString()} and not (${selection} and not (.C or .N)`;
   };

The code is embarrassing, but the latter part shows how to select everything `radius` Å around the `selection`.

**Comment by tibor-postek-m2ms** [comment]
Yes it it distance based selection..... http://nglviewer.org/ngl/api/manual/snippets.html
